src/view/cluster/app/clusterView.tsx

```diff
@@ -0,0 +1,434 @@
+/*-----------------------------------------------------------------------------------------------
+ *  Copyright (c) Red Hat, Inc. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE file in the project root for license information.
+ *-----------------------------------------------------------------------------------------------*/
+
+import * as React from 'react';
+import { makeStyles, Theme, createStyles } from '@material-ui/core/styles';
+import Autocomplete from '@material-ui/lab/Autocomplete';
+import { InsertDriveFile, GetApp, VpnKey, ChevronRight } from '@material-ui/icons';
+import {
+  Avatar,
+  Button,
+  Divider,
+  LinearProgress,
+  List,
+  ListItem,
+  ListItemAvatar,
+  ListItemText,
+  Paper,
+  Stepper,
+  Step,
+  StepLabel,
+  StepContent,
+  TextField,
+  Tooltip,
+  Typography } from '@material-ui/core';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      textAlign: 'left',
+      '& .MuiTextField-root': {
+        margin: theme.spacing(1),
+        width: '25ch',
+      },
+      '& .MuiStepContent-root': {
+        paddingLeft: theme.spacing(5)
+      },
+      '& .MuiStepper-root': {
+        marginLeft: theme.spacing(4)
+      },
+      '& .MuiButton-root': {
+        textTransform: 'none'
+      },
+      '& .MuiStepIcon-root.MuiStepIcon-active': {
+        color: '#BE0000'
+      },
+      '& .MuiStepIcon-root.MuiStepIcon-completed': {
+        color: '#BE0000'
+      },
+      '& .MuiButton-containedPrimary': {
+        backgroundColor: '#BE0000'
+      },
+      '& .MuiStepLabel-iconContainer': {
+        paddingRight: theme.spacing(2)
+      }
+    },
+    button: {
+      marginTop: theme.spacing(1),
+      marginRight: theme.spacing(1)
+    },
+    actionsContainer: {
+      marginBottom: theme.spacing(2),
+      marginTop: theme.spacing(2)
+    },
+    resetContainer: {
+      padding: theme.spacing(3),
+    },
+    formControl: {
+      margin: theme.spacing(1),
+      minWidth: 120,
+      width: '40%'
+    },
+    uploadLabel: {
+      marginTop: theme.spacing(2)
+    },
+    pullSecretField: {
+      marginTop: '20px',
+      width: '100%'
+    }
+  })
+);
+declare global {
+  interface Window {
+      acquireVsCodeApi(): any;
+  }
+}
+
+const vscode = window.acquireVsCodeApi();
+
+const crcVersions = [
+  { crcVersion: "1.0.0", openshiftVersion: "4.2.0"},
+  { crcVersion: "1.1.0", openshiftVersion: "4.2.2"},
+  { crcVersion: "1.2.0", openshiftVersion: "4.2.8"},
+  { crcVersion: "1.3.0", openshiftVersion: "4.2.10"},
+  { crcVersion: "1.4.0", openshiftVersion: "4.2.13"},
+  { crcVersion: "1.5.0", openshiftVersion: "4.2.14"},
+  { crcVersion: "1.6.0", openshiftVersion: "4.3.0"},
+  { crcVersion: "1.7.0", openshiftVersion: "4.3.1"},
+  { crcVersion: "1.8.0", openshiftVersion: "4.3.8"},
+  { crcVersion: "1.9.0", openshiftVersion: "4.3.10"},
+  { crcVersion: "latest", openshiftVersion: "4.4.3"}
```

Comment:
How do I know what is 'latest' 1.9.1 or 1.10.0? There should be version in label.

Reply:
I have used the nomenclature what crc has on mirror.openshift.com. So for this latest is 1.10.0. I can make it this way: 
` { crcVersion: "1.10.0(latest)", openshiftVersion: "4.4.3"}`

Reply:
I have added the latest label to the UI dropdown for crc version select.